docs/core/tools/using-ci-with-cli.md

```diff
@@ -15,93 +15,169 @@ ms.assetid: 0d6e1e34-277c-4aaf-9880-3ebf81023857
-dotnet: 1.0.0-preview2-003121
-```
+In general, a manual setup requires you to acquire a version of the tools (or the latest nightly builds of the tools) and run your build script. You can use a PowerShell or bash script to orchestrate the .NET Core commands or use a project file that outlines the build process. The [orchestration section](#orchestrating-the-build) provides more detail on these options.
 
-Travis can run both `osx` (OS X 10.11) and `linux` ( Ubuntu 14.04 ) job in a build matrix, see [example .travis.yml](https://github.com/dotnet/docs/blob/master/.travis.yml) 
-for more information.
+After you create a script that performs a manual CI build server setup, use it on your dev machine to build your code locally for testing purposes. Once you confirm that the script is running well locally, deploy it to your CI build server. A relatively simple PowerShell script demonstrates how to obtain the .NET Core SDK and install it on a Windows build server:
 
-### AppVeyor
+```powershell
+$ErrorActionPreference="Stop"
+$ProgressPreference="SilentlyContinue"
 
-The [appveyor.com ci](https://www.appveyor.com/) has .NET Core SDK 1.0.1 already installed in the build worker image `Visual Studio 2017`.
+# $LocalDotnet is the path to the locally-installed SDK to ensure the 
+#   correct version of the tools are executed.
+$LocalDotnet=""
+# $InstallDir and $CliVersion variables can come from options to the 
+#   script.
+$InstallDir = "./cli-tools"
+$CliVersion = "1.0.1"
 
-Just use:
+# Test the path provided by $InstallDir to confirm it exists. If it 
+#   does, it's removed. This is not strictly required, but it's a 
+#   good way to reset the environment.
+if (Test-Path $InstallDir)
+{
+    rm -Recurse $InstallDir
+}
+New-Item -Type "directory" -Path $InstallDir
 
-```yaml
-os: Visual Studio 2017
+Write-Host "Downloading the CLI installer..."
+
+# Use the Invoke-WebRequest PowerShell cmdlet to obtain the 
+#   installation script and save it into the installation directory.
+Invoke-WebRequest `
+    -Uri "https://raw.githubusercontent.com/dotnet/cli/rel/1.0.1/scripts/obtain/dotnet-install.ps1" `
+    -OutFile "$InstallDir/dotnet-install.ps1"
+
+Write-Host "Installing the CLI requested version ($CliVersion) ..."
+
+# Install the SDK of the version specified in $CliVersion into the 
+#   specified location ($InstallDir).
+& $InstallDir/dotnet-install.ps1 -Version $CliVersion `
+    -InstallDir $InstallDir
+
+Write-Host "Downloading and installation of the SDK is complete."
+
+# $LocalDotnet holds the path to dotnet.exe for future use by the 
+#   script.
+$LocalDotnet = "$InstallDir/dotnet"
+
+# Run the build process now. Implement your build script here.
+```
+
+You provide the implementation for your build process at the end of the script. When run and after the tooling is acquired, the script executes your build process. For UNIX machines, the following bash script performs the actions described in the PowerShell script in a similar manner:
```

Comment:
nit: "When run and after tooling is acquired" seems awkward. Maybe say "The script acquires the tools and then executes your build process."